packages/postgres-database/tests/test_models_payments_transactions.py

```diff
@@ -71,18 +79,24 @@ async def _init(payment_id: str):
         values["initiated_at"] = utcnow()
 
         # insert
-        await connection.execute(payments_transactions.insert().values(values))
+        await insert_init_payment_transaction(connection, **values)
```

Comment:
please assert the return type, that it is not `PaymentAlreadyExists`

Reply:
I would also add a test for when this fails

Reply:
there is a test about it below: test_complete_transaction